app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= provide :title, "About" %>
+
+<h1>About</h1>
+  <p>
+    <%= link_to "<em>Ruby on Rails Tutorial</em>", "https://www.railstutorial.org/" %>, part of the
+      <%= link_to "Learn Enough", "https://www.learnenough.com/" %> family of tutorials, is a
+        <%= link_to "book", "https://www.railstutorial.org/book" %> and
+          <%= link_to "screencast series", "https://screencasts.railstutorial.org/" %> 
+        to teach web development with
+      <%= link_to "Ruby on Rails", "https://rubyonrails.org/" %>.
+    This is the sample app for the tutorial.
```

Comment:
please, check indenting for those lines

Reply:
Honestly I am not very sure about indenting. But I changed the indent for those line, can you help me check and guide me if it is still wrong